README.md

```diff
@@ -81,3 +81,35 @@ Virtualbox - https://www.virtualbox.org/
 Vagrant - https://www.vagrantup.com/
 
 
+## Docker 
+
+1. To build the image 
+```
+$ docker build --tag todo-app .
+```
+
+2. To run the image 
+``` 
+$ docker run todo-app
+```
+
+3. How to run the app with the bind mount - using a bind mount when running the container makes the project directory on your host machine available as a mounted directory within the container.
+```
+$ docker run --env-file ./.env -p 5000:5000 --mount type=bind,source="$(pwd)"/ todo_app,target=/app/todo_app todo-app 
+```
+
+4. To container can either be run in dev or prod 
+```
+$ docker build --target development --tag todo-app:dev .
+$ docker build --target production --tag todo-app:prod .
+```
+
+5. To get the app to run in the container using the docker-compose.yml
+```
+docker-compose up
```

Comment:
Safest to use `docker-compose up --build`, so it rebuilds the image as necessary.